_includes/samples/PriceFeeds/PriceConsumerV3.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"log"
+
+	proxy "github.com/aelmanaa/chainlink-price-feed-golang/aggregatorv3"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func main() {
+	client, err := ethclient.Dial("https://kovan.infura.io/v3/<infura_project_id>")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Price Feed address
+	addr := "0x9326BFA02ADD2366b30bacB125260Af641031331"
+	aggregatorAddress := common.HexToAddress(addr)
+	instance, err := proxy.NewAggregatorV3Interface(aggregatorAddress, client)
```

Comment:
perhaps we call this instance `chainlinkPriceFeed` just to give us a bit more chainlink mention in the code? Will also help new coders connect the dots with what this represents, especially if they're JS devs that are early to OOP language like "instance" etc.

Reply:
oh , didn't see this comment. yes good idea. Still will add proxy to it ;)